### ISSUE TITLE
Tracks not in order by track number

### ISSUE THREAD

**Issue by zeLostchild** [LibreTime/libretime#470]
<img width="731" alt="screen shot 2018-03-18 at 12 51 12 am" src="https://user-images.githubusercontent.com/37496815/37563983-1e53fc88-2a4a-11e8-85d0-3ac7e9f48843.png">

when organized by album or creator something is breaking the hierarchy and causing it not to organize by track number, this also switched tracks 14<->15, and 16<->17, not pictured (it seems to only switch adjacent numbers). when I specifically select just the album then organize it by track number it organizes it just fine.

another issue that is similar not really a bug though. I know this might be asking without offering anything in return because I have little knowledge on coding/dev. but I would like to know if someone was willing to develop an "album mode" for playing albums like abbey road, without gaps and fluctuating volumes between tracks. Right now I guess I am just turning my albums into large mp3 blocks, and uploading them along side the album with separated tracks. double the space, but no real alternative. 
I'm running Ubuntu 16.04
and downloaded and installed yesterday, so whatever version that is.

**Comment by Robbt** [comment]
Thanks for the feedback, these are definitely aspects that could be improved but they aren't a bug per se. Basically the dataTables jquery library that is used to display tracks sorts primarily based upon what you select at the top and the secondary sort options aren't specified. In an ideal world it would be intelligent about sorting by track number when the other options match but as you point out this isn't the case.

The album mode idea is interesting but volume fluctuation would probably be determined primarily by the source tracks unless you are using replaygain which should in theory attempt to get all tracks to around the same general volume level upon playback. Gaps are determined by the fadeover options in the tracks and there shouldn't generally be gaps inserted by LibreTime.

**Comment by zeLostchild** [comment]
There might not be a volume fluctuation, I am just assuming that there is some sort of volume normalizing going on. There is definitely a gap between playing two songs though, I deleted silan, so cue in / cue out should be exact. I turned off the default fade in - fade out, and crossfade. yet on the tracks "carry that weight" and "the end" off of _abbey road_ album there is definite gap between the two songs. 
I am guessing it has something to do with the interaction between liquidsoap and the commands its getting from libretime. It could also be isolated within liquidsoap and it could be fixed with the liquidsoap configuration. iTunes used to do the same thing a long time ago in a galaxy far far away, but they fixed it. I'm not sure if it is simple fix or complex one.

my idea is if an album mode was that it would turn off crossfade / fade in / fade out for a selected series of songs.
